prov/rxd/src/rxd.h

```diff
@@ -397,29 +415,41 @@ static inline int rxd_match_tag(uint64_t tag, uint64_t ignore, uint64_t match_ta
 }
 
 int rxd_info_to_core(uint32_t version, const struct fi_info *rxd_info,
-		     struct fi_info *core_info);
+			struct fi_info *core_info);
 int rxd_info_to_rxd(uint32_t version, const struct fi_info *core_info,
-		    struct fi_info *info);
+			struct fi_info *info);
 
 int rxd_fabric(struct fi_fabric_attr *attr,
-	       struct fid_fabric **fabric, void *context);
+			struct fid_fabric **fabric, void *context);
 int rxd_domain_open(struct fid_fabric *fabric, struct fi_info *info,
-		    struct fid_domain **dom, void *context);
+			struct fid_domain **dom, void *context);
 int rxd_av_create(struct fid_domain *domain_fid, struct fi_av_attr *attr,
-		  struct fid_av **av, void *context);
+			struct fid_av **av, void *context);
 int rxd_endpoint(struct fid_domain *domain, struct fi_info *info,
-		 struct fid_ep **ep, void *context);
+			struct fid_ep **ep, void *context);
 int rxd_cq_open(struct fid_domain *domain, struct fi_cq_attr *attr,
-		struct fid_cq **cq_fid, void *context);
+			struct fid_cq **cq_fid, void *context);
 int rxd_cntr_open(struct fid_domain *domain, struct fi_cntr_attr *attr,
-		  struct fid_cntr **cntr_fid, void *context);
+			struct fid_cntr **cntr_fid, void *context);
 int rxd_query_atomic(struct fid_domain *domain, enum fi_datatype datatype,
-		     enum fi_op op, struct fi_atomic_attr *attr, uint64_t flags);
+			enum fi_op op, struct fi_atomic_attr *attr, uint64_t flags);
```

Comment:
these indentation fixes --> should they be dropped? or are they alright?

Reply:
These fixes can be dropped. The rule is not never spaces, only tabs. It's tabs where there could be a tab/for every 8 spaces. So these spaces (and a lot of the others you've replaced) are here to lineup the arguments. Those are fine.